src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -176,6 +182,11 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                .WithDescription("If provided, a path to the file that will be used to write the output when using --enable-json-output.")
                .Callback(jsonOutputFile => JsonOutputFile = jsonOutputFile);
 
+            Parser
+               .Setup<string>("runtime")
+               .WithDescription("If provided, determines which version of the host to start.")
```

Comment:
Should the description also provide allowed values for this option?

Reply:
added